tests/func/test_import_url.py

```diff
@@ -90,15 +90,16 @@ def test(self):
 
 
 @pytest.mark.parametrize("dname", [".", "dir", "dir/subdir"])
-def test_import_url_to_dir(dname, repo_dir, dvc_repo):
-    src = repo_dir.DATA
+def test_import_url_to_dir(dname, tmp_dir, dvc):
+    tmp_dir.gen({"data_dir": {"imported_file": "file content"}})
+    src = os.path.join("data_dir", "imported_file")
```

Comment:
But src is not imported, dst is `imported_file`. This is confusing. Also, instead of `filecmp` please use `read_text() == "something"`.